app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3:
+        raise IndexError("list index out of range")
```

Comment:
why do you repeat error? it would be better to explain for user that command is wrong